src/unittests.cfg

```
@@ -22,6 +22,7 @@ secure_mode = False
 hostname_callable = socket:getfqdn
 worker_precheck = False
 default_task_retries = 0
+hide_sensitive_variable_fields = true
```

```suggestion
hide_sensitive_variable_fields = True
```
The other occurrences all use `True` instead of `true`. Nitpicking but nice for consistency.

---

:+1: